Makefile.core.mk

```diff
@@ -101,7 +101,11 @@ check_wasm: build_wasm build_envoy
 clean:
 	@bazel clean
 
-gen: ;
+gen:
+	@scripts/gen-testdata.sh
+
+gen-check:
```

Comment:
Should it be added to "test" or some other top-level make command?

Reply:
We do not really use the generated testdata package in the test targets, so not sure if it makes much sense to add gen check as a dep to them. Though it is already part of presubmit check: https://github.com/istio/proxy/pull/3020/files#diff-eea0e45cbc90a5cc2ef84b472e8cd846, not sure if that addresses your concern?

Reply:
Ah, ok. I missed that script.